lib/project/web-site-project.ts

```diff
@@ -73,7 +77,7 @@ export class MobileWebSiteProject extends frameworkProjectBaseLib.FrameworkProje
 	}
 
 	public adjustBuildProperties(buildProperties: any, projectData?: IProjectData): any {
```

Comment:
return type is any, but we return future? Maybe return type should be IFuture<any> ?